app/config/FrontendAppConfig.scala

```diff
@@ -35,7 +51,7 @@ class FrontendAppConfig @Inject()(val runModeConfiguration: Configuration, envir
     "cymraeg" -> Lang("cy")
   )
 
-  override def mode: Mode.Mode = environment.mode
+//  override def mode: RunMode = environment.mode
```

Comment:
Should this be commented out?

Reply:
I *think* so, but lets keep half an eye out for it as we go through